src/models/searchservice/response/DisplayableFacet.ts

```diff
@@ -22,10 +18,6 @@ export interface DisplayableFacet extends Facet {
  * @public
  */
 export interface DisplayableFacetOption extends FacetOption {
-  /** {@inheritDoc Matcher} */
-  matcher: Matcher,
-  /** {@inheritDoc FacetOption.value} */
-  value: string | number | boolean | NumberRangeValue,
```

Comment:
These are now in `FacetOption`, right?

Reply:
this was the same thing as for `DisplayableFacet` where some properties were being redefined after extending the base model even though nothing was being changed. so I just removed the redundancy